setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
I think this would be clearer:

install_requires = []
with open('requirements.txt') as fh:
    for r in fh:
        r = r.strip()
        if r:
            install_requires.append(r)